concrete/concrete.go

```diff
@@ -167,16 +165,19 @@ func (c *GenericPrecompileRegistry) Precompiles(blockNumber uint64) PrecompileMa
 	return c.precompiles[idx]
 }
 
-func (c *GenericPrecompileRegistry) ActivePrecompiles(blockNumber uint64) []common.Address {
+func (c *GenericPrecompileRegistry) PrecompiledAddresses(blockNumber uint64) []common.Address {
 	idx := c.index(blockNumber)
-
 	if idx < 0 {
 		return []common.Address{}
 	}
 	return c.addresses[idx]
 }
 
 func insert[T any](slice []T, index int, value T) []T {
+	//edge case
```

Comment:
This comment is not useful and it is not properly formatted (missing space, starts in lowercase). Better to just remove it.

Reply:
Done